assignments/objects.js

```diff
@@ -18,47 +18,118 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = {
+  id: '1',
+  email: 'mmelloy0@psu.edu',
+  name: 'Mitzi',
+  gender: 'F'
+}
+
+const kennan = {
+  id: '2',
+  email: 'kdiben1@tinypic.com',
+  name: 'Kennan',
+  gender: 'M',
+  speak() {
+    return `Hello, my name is ${this.name}!`
+  }
+}
+
+const kevin = {
+  id: '3',
+  email: 'kmummery2@wikimedia.org',
+  name: 'Kevin',
+  gender: 'M'
+}
 
+const gannie = {
+  id: '4',
+  email: 'gmartinson3@illinois.edu',
+  name: 'Gannie',
+  gender: 'M'
+}
+
+const antonietta = {
+  id: '5',
+  email: 'adaine5@samsung.com',
+  name: 'Antonietta',
+  gender: 'F',
+  multiplyNums(a, b) {
+    return a * b;
+  }
+}
 
-// ==== Challenge 2: Reading Object Data ==== 
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(mitzi.name);
 
 // Kennan's ID
+console.log(kennan.id);
 
 // Keven's email
+console.log(kevin.email);
 
 // Gannie's name
+console.log(gannie.name);
 
 // Antonietta's Gender
+console.log(antonietta.gender);
 
-// ==== Challenge 3: Object Methods ==== 
+
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+console.log(kennan.speak());
```

Comment:
Can you think of a way to create an object method outside of the object literal?